samples/bluetooth/throughput_many_to_one/README.rst

```diff
@@ -0,0 +1,239 @@
+.. _throughput_many_to_one:
+
+Bluetooth: Throughput many to one
+##########################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Throughput Many-To-One sample demonstrates high-uplink many-to-one transfers over Bluetooth LE Periodic Advertising with Responses (PAwR).
+It is organized as two cooperating applications: an advertiser/host and one or more synchronizer/responders.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf54l15dk_nrf54l15_cpuapp
+
+You need at least two development kits to try the sample (one advertiser and one or more synchronizers). 
+
+Overview
+********
+This sample is a starting point for implementing PAwR for higher-throughput many-to-one communication.
+It consists of two roles:
+
+- ``periodic_adv_rsp``: PAwR advertiser/host that creates a periodic advertising set, encodes control frames (open-slot bitmap, ACKs, and retransmit hints), receives responses from multiple devices, and prints measured and theoretical throughput.
+- ``periodic_sync_rsp``: PAwR synchronizer/responder that discovers and synchronizes to the periodic advertiser, claims a free response slot, and transmits payloads in its assigned slot.
+
+The synchronizer discovers the advertiser by device name (``"PAwR adv sample"``). The advertiser prints measured and theoretical throughput for the active set of responders.
+
+Default parameter values
+========================
+
+.. list-table:: Default parameter values
+   :header-rows: 1
+
+   * - Parameter
+     - Value
+   * - Number of responders (slots)
+     - 2
+   * - Response payload size
+     - 247 bytes
+   * - PHY data rate
+     - LE 2M
+   * - Response slot delay
+     - 3 units (3.75 ms)
+   * - Response slot spacing
+     - computed from payload and PHY
+   * - Subevent interval
+     - 6 units (7.50 ms)
+   * - Retransmit bitmap bytes
+     - up to 4
+
+Limitations and trade-offs
+===========================

```

Comment:
This section could be moved towards the bottom, just before the dependencies section.